lib/cli.js

```diff
@@ -18,7 +18,8 @@ program
   .option("-c, --chdir <path>", "change the working directory")
   .option("-r, --registry <url>", "specify registry url")
   .option("-v, --verbose", "output extra debugging")
-  .option("--no-upstream", "don't use upstream unity registry");
+  .option("--no-upstream", "don't use upstream unity registry")
+  .option("-t", "--test", "add package as testable");
```

Comment:
For the same reason, `-t` should move to the command-add section. See https://github.com/tj/commander.js/issues/521

Reply:
My mistake, I fixed it 😄